MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -23,15 +25,28 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 		viewModel?.loadFeed()
 	}
 
+	@IBAction private func hideErrorMessage() {
+		viewModel?.hideFeedError()
+	}
+
 	func bind() {
 		title = viewModel?.title
 		viewModel?.onLoadingStateChange = { [weak self] isLoading in
 			if isLoading {
+				self?.errorView?.hideMessage()
 				self?.refreshControl?.beginRefreshing()
 			} else {
 				self?.refreshControl?.endRefreshing()
 			}
 		}
+
+		viewModel?.onFeedError = { [weak self] in
+			self?.errorView?.show(message: Localized.Feed.loadError)
+		}
```

Comment:
As a follow up on my previous comments, this block should handle an optional string with an error message and hide the `errorView` if the value is nil
```suggestion
		viewModel?.onErrorStateChange = { [weak self] message in
			if let message = message {
				self?.errorView?.show(message: message)
			} else {
				self?.errorView?.hideMessage()
			}
		}
```